nvidia-install.sh

```diff
@@ -43,8 +50,8 @@ rpm-ostree install \
 
 
 ## nvidia post-install steps
-# disables nvidia repos provided by ublue-os-nvidia-addons
-sed -i 's@enabled=1@enabled=0@g' /etc/yum.repos.d/{eyecantcu-supergfxctl,negativo17-fedora-nvidia,nvidia-container-toolkit}.repo
```

Comment:
This line shouldn't have been removed... only modified...

`nvidia-container-toolkit.repo` should still be disable